lesson_9/cmd/main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"hillel_go_cource/lesson_9/config"
+	computer "hillel_go_cource/lesson_9/models"
+	"hillel_go_cource/lesson_9/server/handlers"
+
+	"github.com/labstack/echo/v4"
+	"github.com/sirupsen/logrus"
+)
+
+func main() {
+
+	pc := computer.NewComputer(
+		"amd ryzen 5600",
+		"amd rx 6650",
+		"MSI B450M",
+		"16",
+		"512",
+		"700",
+		"no",
+	)
+	cfg, err := config.NewConfig()
+	if err != nil {
+		logrus.Fatal(err)
```

Comment:
would be nice to add also custom message like 
`logrus.Fatal("failed to create new config", err)` or something else